3p/3P_CONTRIBUTIONS.md

```diff
@@ -0,0 +1,53 @@
+# Inclusion of third party software into AMP
```

Comment:
We should probably clarify this is for plugins for third-party services, not just third-party software